leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -714,17 +744,41 @@ private boolean checkPlayServices() {
     return true;
   }
 
+  private void setLocationInfo(double lng, double lat) {
+    this.longitude.setText(getString(R.string.longitude) + ": " + lng);
```

Comment:
This starts to be a big methods to hide/show layouts. Please extract them to a new method for a better readability